### ISSUE TITLE
ArgoVocabs test

### ISSUE THREAD

**Issue by danibodc** [nvs-vocabs/test_ArgoVocabs#1]
Test

**Comment by cgourcuf** [comment]
I can't find a way to watch the repo you have created, maybe because it is empty (as it says when I go to this link: ? Could you maybe create a "readme"?

**Comment by mscanderbeg** [comment]
Yes, I had the same experience as Claire - empty repo and can't watch it. Maybe it has to do with how the issue was moved to the dashboard/project?

**Comment by danibodc** [comment]
Thanks @mscanderbeg @cgourcuf, I have added the AVTT team to the repo with write access, and have added a readme file to the repo. Has that made any difference?

**Comment by mscanderbeg** [comment]
The repo is not empty - I see the readme now and I can watch it.  I would guess that it was adding the readme file rather than the write access that helped, but not sure.

**Comment by danibodc** [comment]
Ok good to know!

**Comment by cgourcuf** [comment]
Same, I am now watching the repo.

**Comment by danibodc** [comment]
I've managed to successfully transfer the test repo. The issue card still appears on the AVTT issues management dashboard but points to the new location of the ticket within OneArgo, so that's good! 

Did either of you get any notification of the change?

**Comment by tcarval** [comment]
Hello Danielle, I do not see any transfered issue from AVTT in OneArgo/test_ArgoVocabs, strange

**Comment by danibodc** [comment]
@tcarval this is the only issue that has been transferred currently, we created a test repository containing a single issue ticket (this one) to test the transfer the process.

For me this ticket is still present in the 'To do' column at https://github.com/orgs/nvs-vocabs/projects/16/views/1 - do you see it? 

![image](https://github.com/user-attachments/assets/95584b9c-26e0-49ac-bb3d-9b9e6b1f3aa3)

**Comment by cgourcuf** [comment]
hi @danibodc , yes, we received a notification for the transfer of the repo (I'll forward you the email) and the test issue still appears in the AVTT issues management dashboard, indeed :-)
